### DISCUSSION TITLE
Lanbon L8 with lvgl-haspmota  since update touchscreen no longer works

### DISCUSSION THREAD

**Discussion by waltkind** [arendst/Tasmota#21563] [Support request] [answered]
My Lanbon L8 is now working for 2 month with no great issues.
Now I saw in the change log, that there had been improvements and that the version 14.1 is released.
As the Lanbon needs the PSRAM and the tasmota-lvgl is compiled without I had to compile a new binary.

I did the same as usual in Gitpod, edited the files user_config_override.h and platformio_tasmota_cenv.ini and started. The resulting binary I imported to my Lanbon and the screen looked as usual. But then I realized that the touchscreen doesn't react.
So I tried the 14.0 version with no success. I went back to 13.4.0.4 and it also is not working.

When I switched back to my 13.4.0.4 from april everything is fine.

The differences are:

Program Version         13.4.0.4 (lvgl-haspmota) 
Build Date & Time       2024-04-04T09:26:16 
Core/SDK Version        2_0_14/4.4.6.240105 

Program Version         13.4.0.4 (lvgl-haspmota) 
Build Date & Time       2024-06-04T08:54:09
Core/SDK Version        2_0_15/4.4.7.240312

Does this come from: Removed dedicated touch drivers in favour of Universal Touch driver (#21146)

I would like to have the change:  LVGL fix memory allocation of flush buffers (#21256)

Is there anything I can do?

**Comment by waltkind** [comment]
I went back to version 13.4.0 and the newly compiled version works. 
So I am confident that it is from the removed touch driver and I was lucky to compile my 13.4.0.4 version on april 4th and when it was finaly released on april 15th the driver was removed.

Is there a chance to get the driver back?

**Reply by sfromis** [reply]
Did you use the Universal Display driver already? Doing that has become a requirement for nearly all displays.

**Reply by waltkind** [reply]
I don't know. The display is a ST7789 and I did no changes concerning display for compilation.

**Comment by Jason2866** [comment]
See https://tasmota.github.io/docs/Universal-Display-Driver/

**Reply by waltkind** [reply]
Thank you. 
The display works with no problem. I did an autoconfiguration but in display.ini there was no touch section.
Since the Lanbon has an I2C FT6336U touch controller connected to GPIO4 (SDA) and GPIO0 (SCL) I have no idea what to put there. I tried the Simple resitive touch example but no success.

**Reply by sfromis** [reply]
As FT6336U is not resistive touch (but capacitive), better try what Jason2866 suggested for that specific touch controller chip.

**Comment by Jason2866** [comment, marked as answer]
You could try utouch setting from here https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/ST7789_display.ini

Edit: This settings fit better
```
:UTI,FT6336U,I1,38,*,*
RD A0
CP 02
RTF
RT
:UTT
RDM 00 16
MV 2 1
RT
:UTX
MV 3 2
RT
:UTY
MV 5 2
RT
#
```

**Reply by waltkind** [reply]
Thank you guys. That did the trick.

**Comment by Jason2866** [comment]
Thx, for the feedback will update the Autoconfig
